seed/models/columns.py

```diff
@@ -680,14 +681,14 @@ def _serialize_for_extra_data(column_value):
                 # go through the data and move it to the new field. I'm not sure yet on how long this is
                 # going to take to run, so we may have to move this to a background task
                 orig_data = STR_TO_CLASS[self.table_name].objects.filter(
-                    organization=new_column.organization,
+                    organization=self.organization,
```

Comment:
this doesn't need to be changed, but go ahead and leave it.

Reply:
Yeah, but I figured for consistency I'd go ahead and just reference `self` instead